zokrates_core/src/zir/result_folder.rs

```diff
@@ -0,0 +1,418 @@
+// Generic walk through a typed AST. Not mutating in place
```

Comment:
typed -> zir

Reply:
nice catch, was wrong in the `folder.rs` as well